engine/baml-runtime/src/internal/llm_client/primitive/mod.rs

```diff
@@ -95,13 +100,18 @@ impl TryFrom<(&ClientProperty, &RuntimeContext)> for LLMPrimitiveProvider {
             "anthropic" => {
                 AnthropicClient::dynamic_new(value, ctx).map(LLMPrimitiveProvider::Anthropic)
             }
-            "google-ai" => GoogleClient::dynamic_new(value, ctx).map(LLMPrimitiveProvider::Google),
+            "google-ai" => {
+                GoogleAIClient::dynamic_new(value, ctx).map(LLMPrimitiveProvider::Google)
+            }
+            // "aws-bedrock" => aws::AwsClient::dynamic_new(value, ctx).map(LLMPrimitiveProvider::Aws),
```

Comment:
what happened here?

Reply:
The dynamic_new method is not implemented for AWS right now and AWS was never present here. I just added a commented version of it so it was more explicit. I can either remove it entirely or just explain this there.